scripts/storage-emulator-integration/conformance/firebase.endpoints.test.ts

```diff
@@ -66,178 +69,213 @@ describe("Firebase Storage endpoint conformance tests", () => {
 
   beforeEach(async () => {
     await resetState();
-    await testBucket.upload(smallFilePath, { destination: TEST_FILE_NAME });
   });
 
-  it("should return an error message when uploading a file with invalid metadata", async () => {
-    const fileName = "test_upload.jpg";
-    const errorMessage = await supertest(firebaseHost)
-      .post(`/v0/b/${storageBucket}/o/?name=${fileName}`)
-      .set({ "x-goog-upload-protocol": "multipart", "content-type": "foo" })
-      .expect(400)
-      .then((res) => res.body.error.message);
-
-    expect(errorMessage).to.equal("Invalid Content-Type: foo");
+  describe("multipart upload", () => {
```

Comment:
Are most of these changes to fix the conformance tests?

Reply:
yep, most of these tests were broken when pointed at prod